spec/passaporte_web/service_account_spec.rb

```diff
@@ -116,13 +116,13 @@
     let(:service_account) { PassaporteWeb::ServiceAccount.find("859d3542-84d6-4909-b1bd-4f43c1312065") }
     it "should return the uuid of the ServiceAccount" do
       service_account = PassaporteWeb::ServiceAccount.find("859d3542-84d6-4909-b1bd-4f43c1312065")
-      service_account.account_data.should == {"name" => "Investimentos", "uuid" => "859d3542-84d6-4909-b1bd-4f43c1312065"}
-      service_account.uuid.should == '859d3542-84d6-4909-b1bd-4f43c1312065'
+      expect(service_account.account_data).to eq({"name" => "Investimentos", "uuid" => "859d3542-84d6-4909-b1bd-4f43c1312065"})
+      expect(service_account.uuid).to eq('859d3542-84d6-4909-b1bd-4f43c1312065')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.